src/doc/alerts.xml

```diff
@@ -86,7 +86,8 @@ This list the alerts
 10046	Insecure Component
 10047	HTTPS Content Available via HTTP
 10048	ShellShock
-10049  Image Location Scanner - passive scanner finding
+10049   Image Location Scanner - passive scanner finding
```

Comment:
Not sure if it matters at all but over here you mentioned intent to use 10050: https://github.com/zaproxy/zap-extensions/pull/47#issuecomment-146001555